docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -25,107 +26,32 @@ command:
 Internal SSL/TLS configuration
 ------------------------------
 
-SSL/TLS is configured in the ``config.properties`` file.  The SSL/TLS on the
-worker and coordinator nodes are configured using the same set of properties.
-Every node in the cluster must be configured. Nodes that have not been
-configured, or are configured incorrectly, are not able to communicate with
-other nodes in the cluster.
+The SSL/TLS on the worker and coordinator nodes are configured using the same
+set of properties. Every node in the cluster must be configured. Nodes that have
+not been configured, or are configured incorrectly, are not able to communicate
+with other nodes in the cluster.
 
-To enable SSL/TLS for Trino internal communication, do the following:
+In typical deployments you want to enable :doc:`/security/tls` for access to the
```

Comment:
I’m not sure this is the correct advice. If you have authentication with TLS on the coordinator, you need to enable secure internal communication, otherwise the coordinator needs HTTP enabled and you have to block access with firewall rules. That’s more complicated than enabling both.

Reply:
Its a bit tricky .. I will reword around TLS all the way..